hack/pre-release.sh

```diff
@@ -0,0 +1,72 @@
+# this script updates the WMCO and OCP versions in the Makefile, base CSV, bundle and Dockerfiles.
```

Comment:
What shell is this targeted at?

Please ensure that the script will exit on the first error.